activerecord/lib/active_record/connection_adapters/sqlite3/schema_statements.rb

```diff
@@ -53,6 +53,16 @@ def indexes(table_name)
         end
 
         def add_foreign_key(from_table, to_table, **options)
+          if options[:deferrable] == true
```

Comment:
Rails main branch version is already 7.2.0.alpha, I'd like to keep this deprecation message as the PostgreSQL adapter does.
https://github.com/rails/rails/blob/2a986b77964982607ab146842061b5b5860ccf24/activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb#L526-L528